src/sdlwindow.cpp

```diff
@@ -190,6 +190,12 @@ void inputSDLThreadRun( void )
 						g_nNestedRefresh = g_nOldNestedRefresh;
 						g_bWindowFocused = true;
 						break;
+					case SDL_WINDOWEVENT_EXPOSED:
+						if ( g_bFullscreen == false )
```

Comment:
Shouldn't we always redraw here, regardless of fullscreen state?

Reply:
Yes that'd be a good idea, i don't remember why i added that check there - i did a quick change on my local gamescope repository without it and it seems to work fine. Note though that my repository is months old (286 commits behind) so i don't know if that will interfere with any more recent changes (i'll need to find some time to merge things).